tasks/internal_reconcile_report_orphan/entities/orphan.py

```diff
@@ -0,0 +1,24 @@
+import dataclasses
+from typing import List
+
+
+@dataclasses.dataclass
```

Comment:
Looked into pydantic data classes, but the default constructor uses a dictionary instead of properly named parameters. Might be a future research topic.